loops-conditionals/conditionals/exercise_00.js

```diff
@@ -8,3 +8,26 @@
     7. At the end of the program, print out: "--- Animal check complete ---"
     8. Change the animal to different values to make sure it works in all cases
 */
+
+function animalChecker(animal) {
+    window.console.log('--- Animal Checker Program ---');
+
+    if (animal === 'cat') {
+        window.console.log('Meow!');
+    } else if (animal === 'dog') {
```

Comment:
Good solution. 

Take in account object calisthenics rules in the future 👉 avoid `else` statement.

https://medium.com/@davidsen/clean-code-object-calisthenics-f6f4dec07c8b